mono/mini/aot-compiler.c

```diff
@@ -10477,12 +10547,14 @@ emit_aot_file_info (MonoAotCompile *acfg, MonoAotFileInfo *info)
 		symbols [sindex ++] = "imt_trampolines";
 		symbols [sindex ++] = "gsharedvt_arg_trampolines";
 		symbols [sindex ++] = "ftnptr_arg_trampolines";
+		symbols [sindex ++] = "unbox_arbitrary_trampolines";
 	} else {
```

Comment:
The llvm version needs to be changed too in emit_aot_file_info () in mini-llvm.c.

Reply:
thanks, updated.